product_packaging_multi_barcode_stock_menu/readme/ROADMAP.rst

```diff
@@ -0,0 +1 @@
+In future migration that should be merged in a new module `product_multi_barcode_stock`
```

Comment:
why not renaming this module w/o `_menu` right now? :thinking:

Reply:
done